### DISCUSSION TITLE
part-ii/2019-133/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#28] [Question comments]
Part II 2019 Riemann Surfaces: Paper 1, Section II, F 

https://questions.tripos.org/part-ii/2019-133/

**Comment by artie2000** [comment]
(b): Genus 3.
<details>
<summary>(c):</summary>

$\phi:(x,y)\to (\xi_7^3x,\xi_7y)$

$\pi:(x,y)\to x^2y$

</details>

**Comment by artie2000** [comment]
Is there a nice way to prove that the projections are local homeomorphisms and that the transition maps are analytic, for part (a)?
